microprofile/pom.xml

```diff
@@ -53,6 +53,7 @@
         <module>opentracing-smallrye</module>
         <module>reactive-messaging-smallrye</module>
         <module>reactive-streams-operators-smallrye</module>
+        <module>microprofile-lra</module>
```

Comment:
Can we rename this to "lra" to be consistent with the other MP modules?

Reply:
or lra-narayana?

Reply:
lra-narayana would work too.  Frankly, I was never a huge fan of appending "-smallrye" for the other MP modules - especially since most of those modules do not expose any management configuration, nor any smallrye specific capabilities.